src/components/AboutTeam/useStyles.js

```diff
@@ -0,0 +1,70 @@
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles(({ typography, breakpoints, palette }) => ({
+  root: {
+    backgroundColor: palette.background.paper,
+    padding: `${typography.pxToRem(40)} 0`,
+    [breakpoints.up("md")]: {
+      padding: `${typography.pxToRem(80)} 0`,
+    },
+  },
+  cardRoot: {
+    boxShadow: "none",
+    backgroundColor: "unset",
+    padding: `0 ${typography.pxToRem(20)}`,
+    [breakpoints.up("md")]: {
+      marginRight: typography.pxToRem(16),
+      padding: 0,
+    },
+  },
+  cardMedia: {
+    height: typography.pxToRem(278),
```

Comment:
Looking at the XD, the image is limited to width of 278 even in mobile

Reply:
Fixed